contribuer/pages/index.js

```diff
@@ -1,55 +1,619 @@
+                périodicité (ponctuelle, mensuelle, annuelle, autre), la légende
+                (ex.&nbsp;: “500&nbsp;€ maximum”) et le montant maximal.
+              </p>
+            </div>
+          </details>
+
+          <details>
+            <summary>
+              Limiter l’affichage selon un intérêt particulier&nbsp;?
+            </summary>
+            <div className="details-content">
+              <p>
+                Oui, si nécessaire. Vous pouvez cibler un public intéressé par
+                le BAFA, le permis de conduire, les études à l’étranger, ou les
+                formations du sanitaire et social.
+              </p>
+            </div>
+          </details>
+
+          <p className="top-link">
+            <a href="#top">↑ Retour en haut</a>
+          </p>
+        </section>
       </main>
+
+      <style jsx>{`
+        .container {
+          max-width: 960px;
+          margin: 0 auto;
+          padding: 2rem 1rem 4rem;
+          line-height: 1.65;
+        }
+        h1 {
+          font-size: 2rem;
+          margin-bottom: 0.5rem;
+        }
+        h2 {
+          font-size: 1.5rem;
+          margin: 1.5rem 0 0.75rem;
+        }
+        h3 {
+          font-size: 1.15rem;
+          margin: 1.25rem 0 0.5rem;
+        }
+        .toc {
+          background: #f6f8fb;
+          border: 1px solid #e6e8eb;
+          border-radius: 10px;
+          padding: 1rem 1.25rem;
+          margin: 1rem 0 2rem;
+        }
+        .toc ol {
+          margin: 0.25rem 0 0.25rem 1.25rem;
+        }
+        .cta {
+          background: #fff;
+          border: 1px solid #e6e8eb;
+          border-radius: 10px;
+          padding: 1rem 1.25rem;
+          margin: 1.5rem 0 2rem;
+        }
+        .button {
+          display: inline-block;
+          padding: 0.5rem 0.9rem;
+          background: #0a66ff;
+          color: #fff;
+          border-radius: 8px;
+          text-decoration: none;
+          font-weight: 600;
+        }
+        .video-item {
+          list-style: none;
+          margin-top: 0.5rem;
+        }
+        video {
+          width: 100%;
+          max-width: 720px;
+          height: auto;
+          margin-top: 0.75rem;
+          border-radius: 8px;
+          border: 1px solid #e6e8eb;
+        }
+        section {
+          margin-top: 1rem;
+        }
+        ul,
+        ol {
+          padding-left: 1.25rem;
+        }
+        a {
+          color: #0a66ff;
+          text-decoration: none;
+        }
+        a:hover,
+        a:focus {
+          text-decoration: underline;
+        }
+        details {
+          background: #fff;
+          border: 1px solid #e6e8eb;
+          border-radius: 10px;
+          padding: 0.75rem 1rem;
+          margin: 0.75rem 0;
+        }
+        details summary {
+          cursor: pointer;
+          font-weight: 600;
+          outline: none;
+        }
```

Comment:
Un peu de style pour la route 🕺 
```suggestion
        }
        .top-link {
          margin-top: 3rem;
          margin-bottom: 0rem;
          text-align: center;
        }
```